README.md

```diff
@@ -70,11 +70,11 @@ Invoke is used to run a function repeatedly with multiple inputs. It gives you m
 Example [invoke_echo.py](https://github.com/capeprivacy/pycape/blob/main/examples/invoke_echo.py):
 
 ```py
-from pycape import Cape
+from pycape import Cape, FunctionRef
```

Comment:
Nice catch for the missing import! 
`nit`: can you just have the import on separate lines? we try to follow [this guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#313-imports-formatting):
```
from pycape import Cape
from pycape import FunctionRef
```
Thank you

Reply:
@miiklay note this can be done automatically by running `make fmt` from the project root